CPC-CHARTER.md

```diff
@@ -233,51 +233,33 @@ election is required if there is only one candidate and no objections to
 the candidates election. Elections shall be done within the Projects by
 the Collaborators active in the Project.
 
+The CPC will elect from amongst CPC members:
 
-The CPC will elect from amongst CPC members a CPC Chairperson to
-work on building an agenda for CPC meetings and a CPC Director to represent
-the CPC to the Board for a term of one year according to the OpenJS Foundation’s
-By-laws. The Chair and Director may be (but are not required to be)
-the same person. The CPC shall hold annual elections to select a CPC
-Chairperson and Director; there are no limits on the number of terms a
-CPC Chairperson or Director may serve.
+* a CPC Chairperson to work on building an agenda for CPC meetings
+* CPC Directors to represent the Foundation's projects and related communities to the Board
 
-## Section 8. Board Representation
+The term for each of these roles is one year (as defined in the [OpenJS Foundation bylaws][]).
 
-There are 3 Board seats earmarked for representation from the
-OpenJS Foundation, the Foundation's projects, and related communities.
+The CPC shall hold annual elections to select a CPC Chairperson and Directors.
+The CPC can decide to spread those elections across the year in order to avoid changing all of its Board representatives at once.
 
-### The Primary CPC Director (as defined in §4.3(d) in the [OpenJS Foundation bylaws][])
+There are no limits on the number of terms a CPC Chairperson or Director may serve.
 
-The first of these board seats is filled as outlined in Section 7 on [Elections][] and is always in place.
+The Chair may be (but is not required to be) a CPC Director.
 
-**Primary CPC Director eligibility:** The Primary CPC Director must be a current CPC Member
 
-**Primary CPC Director voting:** CPC Voting Members will vote on candidates
+## Section 8. Board Representation (CPC Directors)
 
-### The Secondary CPC Director (as defined in §4.3(e) in the [OpenJS Foundation bylaws][])
+There are Board seats allocated for CPC members to represent the Foundation's projects and related communities to the Board.
```

Comment:
How many? We should be precise here.

Reply:
The PR seems to defer to the bylaws for the count, which seems good because then there's one source of truth?

Reply:
@ljharb yeah, that's the idea indeed. It says so a couple of lines below. I welcome suggestions as to how to make that more clear.